src/pages/UserGroups/GroupCard.tsx

```diff
@@ -32,10 +31,10 @@ export default function GroupCard({ groupData }: { groupData: Group }) {
               />
             ))}
           </div>
-        </div>
+        </Row>
         {/* Open group button */}
         <button
-          className="w-full rounded-lg border border-gray-100 bg-brand_off_white py-3 text-sm font-semibold text-secondary shadow-sm1"
+          className="shadow-md2 w-full rounded-lg border border-gray-100 bg-brand_off_white py-3 text-sm font-semibold text-secondary"
```

Comment:
add `shadow-sm1` back in tailwind css and replace this with `shadow-sm1`

Reply:
shadow should be 4px 20px, which will be md2